CHANGELOG.md

```diff
@@ -26,8 +22,14 @@ PLEASE REMEMBER TO CHANGE THE '..main' WITH AN ACTUAL TAG in GITHUB LINK.
   * {func}`jax.scipy.special.expit` and {func}`jax.scipy.special.logit` now
     require their arguments to be scalars or JAX arrays. They also now promote
     integer arguments to floating point.
+  * The `DeviceArray.tile()` method is deprecated, because numpy arrays do not have a
+    `tile()` method. As a replacement for this, use {func}`jax.numpy.tile`
+    ({jax-issue}`#10266`).
 
 ## jaxlib 0.3.7 (April 15, 2202)
```

Comment:
Typo: `2202` -> `2022`

Reply:
Thanks, added this fix to https://github.com/google/jax/pull/10324